string source = rogr
string target = distancesensor
int sensor_id 
time timestamp

